src/modules/fileActivityMonitor.ts

```diff
@@ -45,7 +46,8 @@ async function handleFileSave(uri: vscode.Uri) {
 
   const config = fileService.getConfig();
   if (config.uploadOnSave) {
-    const fspath = uri.fsPath;
+    const fspath = await realpathSync.native(uri.fsPath);
```

Comment:
Ok, it seems that I had to update the "@types/node" npm dependency to at least v9.2.0 because the `.native(path[, options])` property wasn't available before.

Reply:
@Natizyskunk 
Sorry, This is my mistake.
As I worked on several branches, the node_modules folder remained updated, so I didn't get an error
😅